### PR TITLE
Update documentation about CamelSources

### PR DESCRIPTION
**Pull request by nicolaferraro**
## Proposed Changes

- Align CamelSource doc to changes in upcoming 0.10 release (https://github.com/knative/eventing-contrib/pull/680)

### CODE REVIEW

File: docs/eventing/samples/apache-camel-source/README.md [hunk 1]
```diff
@@ -52,19 +50,21 @@ Deploy the [`display_resources.yaml`](./display_resources.yaml):
 kubectl apply --filename display_resources.yaml
 ```
 
-### Run a CamelSource using the Timer component
+### Run a "Timer" CamelSource
```
[the remark this answers is not in the document]
Comment (samodell) [reply docs/eventing/samples/apache-camel-source/README.md]: I think we want to avoid stylizing the particular CamelSource with quotation marks.

**Comment by nicolaferraro** [comment]
Thanks @samodell, I've applied your suggestions and the content looks better now :)

**Pull request merged**